src/main/java/com/example/clone/service/JwtService.java

```diff
@@ -0,0 +1,72 @@
+package com.example.clone.service;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import java.security.Key;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Component
+public class JwtService {
+
+    public static final String SECRET = "5367566B59703373367639792F423F4528482B4D6251655468576D5A71347437";
```

Comment:
The secret must not be stored as a string, use byte array

Reply:
I stored it in the app properties as I am unsure as to what you meant by byte array, I can make it into a byte array but it would still be unsafe within the service file, unless you meant for me to store it in a byte array and also move it to some config?